src/__swaps__/utils/numbers.ts

```diff
@@ -314,7 +314,8 @@ export const convertBipsToPercentage = (value: BigNumberish, decimals = 2): stri
 export const convertAmountToNativeDisplayWorklet = (
   value: number | string,
   nativeCurrency: keyof nativeCurrencyType,
-  useThreshold = false
+  useThreshold = false,
+  ignoreAlignment = false
```

Comment:
for the swap native input values, we want to keep the native currency symbol on the left (ignore alignment) so that the user can easily update the value